content/pt/docs/what-is-opentelemetry.md

```diff
@@ -0,0 +1,133 @@
+---
+title: O que é OpenTelemetry?
+description:
+  Uma breve explicação sobre o que é o OpenTelemetry e o que ele não é.
+weight: 150
+default_lang_commit: f17277371622b97df4c15a6cfe3ce0b22e5538ef
+---
+
+O OpenTelemetry é uma estrutura de trabalho (_framework_) e um conjunto de
+ferramentas de
+[Observabilidade](/docs/concepts/observability-primer/#what-is-observability)
+projetados para criar e gerenciar dados de telemetria como
+[rastros](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) e
+[logs](/docs/concepts/signals/logs/). Essencialmente, o OpenTelemetry é
+agnóstico em relação a fornecedores e ferramentas, o que significa que pode ser
+utilizado com uma grande variedade de sistemas (_backends_) de observabilidade,
+incluindo ferramentas de código aberto como
+[Jaeger](https://www.jaegertracing.io/) e [Prometheus](https://prometheus.io/),
+além de soluções comerciais.
+
+O OpenTelemetry não é um backend de observabilidade como Jaeger, Prometheus ou
+outra soluções comerciais. O OpenTelemetry é focado na geração, coleta e
+exportação de telemetria. Um dos principais objetivos do OpenTelemetry é
+permitir que você possa instrumentar suas aplicações ou sistemas de forma fácil,
+independentemente da linguagem, infraestrutura ou ambiente de execução. Em
+geral, o armazenamento e a visualização da telemetria são intencionalmente
+deixados para outras ferramentas.
+
+## O que é observabilidade?
+
+[Observabilidade](/docs/concepts/observability-primer/#what-is-observability) é
+a capacidade de entender o estado interno de um sistema ao examinar suas saídas.
```

Comment:
```suggestion
a capacidade de entender o estado interno de um sistema ao examinar suas saídas.
```

"Examinar suas saidas" sounds really strange to me (like the system is crashing/existing) 😅. Maybe something like this reads better in Portuguese.

```suggestion
a capacidade de entender o estado interno de um sistema ao examinar o dados que ele emite.
```
Even though it deviates a bit from the original English text, I found "emits" is better than "saidas" which translates to "exists" in English. We even use "emitir" down below which connects nicely here.

Reply:
I like @joaopgrassi suggestion: `os dados que ele emite`